src/App.jsx

```diff
@@ -1,20 +1,22 @@
 import logo from "/logo.png";
 import "./App.css";
+import Clock from "./clock.jsx"
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
+
   return (
-    <>
+    <div>
       <div>
         <img src={logo} className="logo" alt="Rocket logo" />
       </div>
-      <h1>World Clock</h1>
-      <div className="card">
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-    </>
+          <h1>WORLD CLOCK</h1> 
+          <Clock city="Singapore" timeZone="Singapore"/>
+          <Clock city="Japan" timeZone="Japan"/> 
+          <Clock city="Los Angeles" timeZone="America/Los_Angeles"/>
```

Comment:
By using an array of objects with city and timeZone keys, we could run a single map and don't need to repeat the Clock component over and over, and are able to easily extend our application